server/src/models/trip.model.ts

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('./index.ts');
+
+const Schema = mongoose.Schema;
+
+const tripSchema = new mongoose.Schema({
```

Comment:
We may change some of these fields - currently I have name and country in the front end, but that can be done later when we have a better idea of the data model